Tabloid/client/src/modules/tagManager.js

```diff
@@ -18,3 +18,16 @@ export const getAllTags = () => {
         });
     });
 };
+
+export const createTag = (tag) => {
+    return getToken().then((token) =>
+      fetch(_apiUrl, {
+        method: "POST",
+        headers: {
+          Authorization: `Bearer ${token}`,
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(tag),
+      }).then((resp) => resp.json())
```

Comment:
Should there be any error handling here?

Reply:
added error handling.